src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.c

```diff
@@ -665,174 +719,179 @@ void func_80AEACF8(EnLiftNuts* this, PlayState* play) {
                 } else {
-                    this->unk_354++;
+                    this->frames++;
                 }
                 break;
 
             default:
                 break;
         }
     }
-    func_80AEB974(this);
+    EnLiftNuts_SetEyeTexIndex(this);
 }
 
-void func_80AEAEAC(EnLiftNuts* this) {
+void EnLiftNuts_SetupMove(EnLiftNuts* this) {
     this->actor.speed = 2.0f;
-    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, 1);
-    func_80AE9AC4(this, 1);
-    func_80AE9B4C(1, 1);
-    this->actionFunc = func_80AEAF14;
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, ENLIFTNUTS_ANIM_WALK);
+    EnLiftNuts_AutoTalk(this, ENLIFTNUTS_AUTOTALK_MODE_SET_ON);
+    EnLiftNuts_MinigameState(ENLIFTNUTS_MINIGAME_STATE_MODE_SET, ENLIFTNUTS_MINIGAME_STATE_STARTING);
+    this->actionFunc = EnLiftNuts_Move;
 }
 
-void func_80AEAF14(EnLiftNuts* this, PlayState* play) {
+void EnLiftNuts_Move(EnLiftNuts* this, PlayState* play) {
     f32 dist;
 
     Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.home.rot.y, 10, 0x1000, 0x500);
-    dist = Math_Vec3f_StepTo(&this->actor.world.pos, &this->vec_1D8, this->actor.speed);
+    dist = Math_Vec3f_StepTo(&this->actor.world.pos, &this->waypointPos, this->actor.speed);
     this->actor.world.pos.y += this->actor.gravity;
 
     if (dist == 0.0f) {
-        func_80AEAF8C(this);
+        EnLiftNuts_SetupMovePlayer(this);
     }
 }
 
-void func_80AEAF8C(EnLiftNuts* this) {
-    this->actionFunc = func_80AEAFA0;
+void EnLiftNuts_SetupMovePlayer(EnLiftNuts* this) {
+    this->actionFunc = EnLiftNuts_MovePlayer;
 }
 
-void func_80AEAFA0(EnLiftNuts* this, PlayState* play) {
+void EnLiftNuts_MovePlayer(EnLiftNuts* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
     f32 dist;
-    f32 temp_fv0;
+    f32 magnitude;
+    s16 playerYaw;
     s16 yaw;
-    s16 sp28;
     s16 yawDiff;
 
-    sp28 = this->actor.yawTowardsPlayer - 0x8000;
-    yaw = Math_Vec3f_Yaw(&player->actor.world.pos, &this->actor.home.pos);
-    yawDiff = yaw - sp28;
+    yaw = this->actor.yawTowardsPlayer - 0x8000;
+    playerYaw = Math_Vec3f_Yaw(&player->actor.world.pos, &this->actor.home.pos);
+    yawDiff = playerYaw - yaw;
     dist = Math_Vec3f_DistXZ(&player->actor.world.pos, &this->actor.home.pos);
 
     if (this->actor.xzDistToPlayer < dist) {
         if (ABS_ALT(yawDiff) < 0x2000) {
-            yaw = (yawDiff > 0) ? (yaw + 0x2000) : (yaw - 0x2000);
+            playerYaw = (yawDiff > 0) ? (playerYaw + 0x2000) : (playerYaw - 0x2000);
         }
     }
+
     if (dist < 5.0f) {
-        temp_fv0 = 10.0f;
+        magnitude = 10.0f;
     } else if (dist < 30.0f) {
-        temp_fv0 = 40.0f;
+        magnitude = 40.0f;
     } else {
-        temp_fv0 = 80.0f;
+        magnitude = 80.0f;
     }
 
     play->actorCtx.unk268 = 1;
-    func_800B6F20(play, &play->actorCtx.unk_26C, temp_fv0, yaw);
+    func_800B6F20(play, &play->actorCtx.unk_26C, magnitude, playerYaw);
```

Comment:
`actorCtx.unk268` and `actorCtx.unk_26C` are interesting. Together, they seem to overwrite controller input for player for a frame. `unk268` is a boolean to signal to player to read the input from `unk_26C` once, and `unk268` is turned off every frame after player input is processed.

Could at least make `unk268 = true;`